### ISSUE TITLE
I cannot drag my connections to other connection panels

### ISSUE THREAD

**Issue by sandersaares** [mRemoteNG/mRemoteNG#958]
## Expected Behavior
I grab the tab header of a connection and drag-and-drop it to another connection panel to give it a new home.

## Current Behavior
I grab the tab header of a connection and drag-and-drop it but all it does is reorder the list of tabs. As far as I can tell it is impossible to actually move any connections to the new connection panel I created.

On the below screenshot you can see my empty new connection panel on the right, sitting there sadly with no connections because I do not seem to have any way to put connections in it.

![image](https://user-images.githubusercontent.com/9914262/39032034-64b454ae-4474-11e8-9495-b5262d27953e.png)

## Steps to Reproduce (for bugs)

1. Make two connections, as two tabs.
2. Make a new connection panel.
3. Try to drag and drop a connection to this panel.

## Context
This is my first time using mRemoteNG. It looks very convenient at first glance but this basic feature not working really left a sour taste. I hope you fix it soon!

## Your Environment

* Version used:  v1.75.7012
* Operating System and version (e.g. Windows 10 1709 x64): Windows 10 1709 x64

**Comment by sanderdegroot** [comment]
I don't believe moving connections to a different connection panel is possible after connecting; however, it is possible to choose the connection panel prior to connecting.

First, create one or more connection panels from the View menu > Add Connection Panel. These can be dragged around and docked.

Next, right-click a connection in the Connections panel and select Connect (with options) > Choose panel before connecting. This will allow you to place your connection into one of the connection panels you created earlier.

I would like to see moving connections after they have been connected implemented, but I'm not sure how complex it would be to do so.

**Comment by donatelloOo** [comment]
This would be extremely useful to move connections tab to another panel after connection (using context menu or drap/drop).
That's maybe the main "feature request" to work on for my everyday usage.
NB: please edit the title with prefix "Feature request: " to normalize the name.

That's like if you have several screens, you have to choose on which one to open a document before starting to work... and cannot change your mind after that ! That's make no sens in my point of view.
